oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/VersionGarbageCollector.java

```diff
@@ -1007,6 +1008,10 @@ public void collectGarbage(final NodeDocument doc, final GCPhases phases) {
                 // here the node is not orphaned which means that we can reach the node from root
                 collectDeletedProperties(doc, phases, op, traversedState);
                 switch(revisionDetailedGcType) {
+                    case NO_OLD_PROP_REV_GC : {
+                        // this mode does neither unusedproprev, nor unmergedBC
```

Comment:
I would add a INFO log here.

Reply:
this would log it quite frequently no? - what about doing it in the constructor?

Reply:
PS: created OAK-10727 to follow-up on this